src/js/locale/pl.yaml

```diff
@@ -0,0 +1,532 @@
+name: Polski
+
+##
+# Common and reusable phrases
+##
+errors:
+  uri_not_found: 'Nie rozpoznano URI %{uri}'
+  could_not_load: 'Nie można załadować %{name}'
+  could_not_load_library: 'Nie można załadować %{name} od %{provider}'
+  no_results: Nic nie znaleziono
+  authorization_required: 'Musisz najpierw zautoryzować %{provider}'
+  enable_first: 'Odblokuj najpierw %{provider}'
+  no_service_worker: Nie wspierany service worker
+  required: Wymagane
+  unknown_error: Nieznany błąd
+  need_to_be_online: Musisz być online by załadować ten zasób
+actions:
+  play: Odtwarzaj
+  play_all: Odtwarzaj wszystkie
+  play_next: Odtwarzaj następny
+  shuffle_play: Odtwarzaj losowo
+  pause: Pauza
+  stop: Stop
+  add_to_playlist: Dodaj do playlisty
+  add_to_library: Dodaj do biblioteki
+  remove_from_library: Usuń z biblioteki
+  add_to_queue: Dodaj do kolejki
+  reset: Reset
+  back: Cofnij
+  refresh: Odśwież
+  enable: Odblokuj
+  disable: Zablokuj
+  send: Wyślij
+  save: Zapisz
+  start: Start
+  edit: Edytuj
+  add: Dodaj
+  confirm: Czy na pewno?
+  delete: Usuń
+  remove: Usuń
+common:
+  popularity: Popularność
+  name: Nazwa
+  artist: Wykonawca
+  duration: Długość
+  source: Źródło
+  sources: Źródła
+  sort: Sortuj
+  filter: Filtruj
+  by: ' po '
+fields:
+  start_typing: 'Pisz tutaj...'
+  search: 'Szukaj...'
+  sort: Sortuj
+  view: Widok
+  source: Źródło
+  uri: URI
+  items_to_add:
+    label: Do dodania
+    placeholder: Nic tu nie ma
+  filter: Filtruj
+  filters:
+    all: Wszystkie
+    thumbnails: Ikony
+    list: Lista
+    as_loaded: Jako załadowane
+    name: Nazwa
+    artist: Wykonawca
+    updated: Zaktualizowano
+    tracks: Utwory
+    source: Żródło
+    editable: Edytowalny
+    owner: Właściciel
+    followers: Obserwujący
+    popularity: Popularność
+time:
+  seconds: 
+    short: sek
+    singular: sekunda
+    plural: sekund
+  minutes: 
+    short: min
+    singular: minuta
+    plural: minut
+  hours: 
+    short: gdz

```

Comment:
According to Polish Wikipedia page:
> Godzina, skrót godz., oznaczenie h (ang. hour)

Use either `godz` or `h`, I prefer the latter.